scripts/lochness_create_template.py

```diff
@@ -232,26 +233,31 @@ def create_config_template(config_loc: Path, args: object) -> None:
         for study in args.studies:
             line_to_add = f'''
     {study}:
-        namespace: /DATA/ROOT/UNDER/BOX
+        namespace: /DATA/ROOT/UNDER/MEDIAFLUX
         delete_on_success: False
         file_patterns:
             actigraphy:
                 - vendor: Philips
                   product: Actiwatch 2
-                  data_dir: all_BWH_actigraphy
-                  pattern: 'accel/*csv'
+                  data_dir: actigraphy
+                  pattern: '*csv'
                   protect: True
                 - vendor: Activinsights
                   product: GENEActiv
-                  data_dir: all_BWH_actigraphy
-                  pattern: 'GENEActiv/*bin,GENEActiv/*csv'
+                  data_dir: actigraphy
```

Comment:
Is it mentioned anywhere that a user needs to adjust the patterns if remote patterns are different from the ones created by `lochness_create_template.py`?

Reply:
only briefly mentioned in `lochness/README.md` as below
> Change information in config.yml and lochness.json as needed.

For Pronet & Prescient, I'll be editing the `config.yml` with them to make sure.